server/ec2spotmanager/Provider.py

```diff
@@ -0,0 +1,13 @@
+class Provider():
```

Comment:
This doesn't need to be it's own thing. Why not make it a static method of `CloudProvider`?

Reply:
removed the provider module and moved it into CloudProvider.

Reply:
still needs to be removed. `git rm` ?